### DISCUSSION TITLE
백신종류

### DISCUSSION THREAD

**Discussion by SGLEEKOR** [changdoc/naver-vaccine-macro#275] [General]
안녕하세요 개발자님
감사합니다.
혹시 프로그램에서 화이자 백신 선택하고 해당 병원을 열었어도 아래 첨부그림과 같이 백신종류에 "선택해주세요"라고 나오나요?

![image](https://user-images.githubusercontent.com/86994768/129650218-f2a60574-ce9f-48d4-ba20-c97ede23922e.png)

**Comment by changdoc** [comment]
네 잔여 백신이 발생한 상황에서는 백신 종류 별로 잔여 수량이 노출되게 됩니다.
